cypress/e2e/brewery/SolutionUnauthorized.cy.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import { Login, Workspaces, ErrorBanner } from '../../commons/actions';
+import { stub } from '../../commons/services/stubbing';
+import { GENERIC_SELECTORS } from '../../commons/constants/generic/IdConstants';
+import { EXTENDED_WORKSPACES_LIST } from '../../fixtures/stubbing/default';
+import { apiUtils } from '../../commons/utils';
+
+describe('User has no access to the solution', () => {
+  stub.solutionError = {
+    statusCode: 403,
+    body: {
+      title: 'Forbidden',
+      status: 403,
+      detail: `RBAC SOL-unauthorized - User does not have permission read`,
+    },
+  };
+
+  before(() => {
+    stub.start();
+  });
+
+  after(() => {
+    stub.stop();
+  });
+
+  const customSelectWorkspace = (workspaceId) => {
+    const queries = [apiUtils.interceptPowerBIAzureFunction(), apiUtils.interceptGetScenarios()];
```

Comment:
Maybe we should add a comment here to document that we removed the solution interception from the "usual" queries, and it is replaced by a custom solution interception in the test body